shell/platform/fuchsia/flutter/platform_view.cc

```diff
@@ -73,7 +73,7 @@ PlatformView::PlatformView(
     OnCreateSurface on_create_surface_callback,
     std::shared_ptr<flutter::ExternalViewEmbedder> external_view_embedder,
     fml::TimeDelta vsync_offset,
-    zx_handle_t vsync_event_handle)
+    SessionConnection* session_connection)
```

Comment:
Lets be good about tracking lifetimes and make this a shared_ptr

Reply:
Yeah, I should've marked this as TODO.

I left it as a raw pointer until I have a FakeSessionConnection class available to use for platform_view_unittests.